DOCS-2817 and DOCS-2823: Change Micro-RDK to viam-micro-server (#3266)
viamrobotics/docs: docs/appendix/glossary/viam-micro-server.md

---
title: viam-micro-server
id: viam-micro-server
full_link: /architecture/#viam-server-and-viam-micro-server
short_description: The lightweight version of viam-server that can run on ESP32 devices.
---

The lightweight version of `viam-server`, built for microcontrollers.

> `viam-micro-server` is the open-source executable binary that runs on your microcontroller and provides Viam functionality to your machine.

Reviewer: ```suggestion
`viam-micro-server` is a set of open-source utilities which run on your microcontroller and provides Viam functionality to your machine.
```
It's not a single binary. 

"There is an installer program, a linkable library, an FFI library, a server (which may or may not have an invocable binary), etc. So, we need a name that encompasses them all."

Quoting Andrew Morrow
